salt/metalk8s/addons/prometheus-adapter/deployed/chart.sls

```diff
@@ -299,11 +300,11 @@ metadata:
     app.kubernetes.io/managed-by: salt
     app.kubernetes.io/name: prometheus-adapter
     app.kubernetes.io/part-of: metalk8s
-    chart: prometheus-adapter-2.3.1
+    chart: prometheus-adapter-2.10.1
     heritage: metalk8s
     release: prometheus-adapter
   name: prometheus-adapter-auth-reader
-  namespace: metalk8s-monitoring
+  namespace: kube-system
```

Comment:
Why would this RoleBinding need to live in `kube-system`?

Reply:
Answering my own question: https://github.com/prometheus-community/helm-charts/commit/1a2157dae1ed1382ae7e54871990f8629cfe5ef0